pkg/manifests/manifests_test.go

```diff
@@ -3113,6 +3113,21 @@ func TestAlertManagerUserWorkloadConfiguration(t *testing.T) {
 	if a.Spec.ExternalURL != expectedExternalURL {
 		t.Fatalf("Alertmanager external URL is not configured correctly, expected %s, but got %s", expectedExternalURL, a.Spec.ExternalURL)
 	}
+
+	if !slices.ContainsFunc(a.Spec.Volumes, func(v v1.Volume) bool {
```

Comment:
ci/prow/unit failed
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cluster-monitoring-operator/2399/pull-ci-openshift-cluster-monitoring-operator-release-4.15-unit/1807686194448830464
```
 pkg/manifests/manifests_test.go:3117:13: undefined: slices.ContainsFunc
pkg/manifests/manifests_test.go:3126:13: undefined: slices.ContainsFunc 
```
OCP 4.15 used go1.20, slices.ContainsFunc is since go1.21 if I'm not wrong